packages/dashboard/src/app/views/shared/savedTemplateGridView.js

```diff
@@ -129,39 +120,38 @@ const SavedTemplateGridView = ({
                 itemRefs.current[template.id] = el;
               }}
             >
-              <FocusableGridItem
+              <button
                 onFocus={() => {
                   setActiveGridItemId(template.id);
                 }}
-                isSelected={isActive}
                 tabIndex={tabIndex}
                 title={sprintf(
                   /* translators: %s: story title.*/
                   __('Press Enter to explore details about %s', 'web-stories'),
                   template.title
                 )}
               />
-              <CardPreviewContainer
-                ariaLabel={sprintf(
-                  /* translators: %s: story title. */
-                  __('Preview of %s', 'web-stories'),
-                  template.title
-                )}
-                tabIndex={tabIndex}
-                pageSize={pageSize}
-                story={template}
-                centerAction={{
-                  targetAction: (e) => actions.previewTemplate(e, template),
-                  label: centerActionLabelByStatus[template.status],
-                }}
-                // this doesn't work yet and will throw an error in the editor, commenting out to prevent headaches while testing UI but leaving code in so that we know where things attach
-                // bottomAction={{
-                //   targetAction: bottomTargetAction(template),
-                //   label: bottomActionLabel,
-                // }}
+              <div
+              // ariaLabel={sprintf(
+              //   /* translators: %s: story title. */
+              //   __('Preview of %s', 'web-stories'),
+              //   template.title
+              // )}
+              // tabIndex={tabIndex}
+              // pageSize={pageSize}
+              // story={template}
+              // centerAction={{
+              //   targetAction: (e) => actions.previewTemplate(e, template),
+              //   label: centerActionLabelByStatus[template.status],
+              // }}
+              // this doesn't work yet and will throw an error in the editor, commenting out to prevent headaches while testing UI but leaving code in so that we know where things attach
+              // bottomAction={{
+              //   targetAction: bottomTargetAction(template),
+              //   label: bottomActionLabel,
+              // }}
               />
```

Comment:
IMO we should take this out. We can always come back later to look in our git history if we want this 😄

Reply:
yup! Ticket in backlog, i'll grab it after I'm out of the dashboard a bit to avoid keeping branches current. https://github.com/google/web-stories-wp/issues/8746